src/renderer/components/+catalog/catalog.tsx

```diff
@@ -151,7 +157,7 @@ export class Catalog extends React.Component {
         provideBackButtonNavigation={false}
         contentGaps={false}>
         <ItemListLayout
-          renderHeaderTitle={this.catalogEntityStore.activeCategory?.metadata.name}
+          renderHeaderTitle={this.catalogEntityStore.activeCategory?.metadata.name ?? "Browse All"}
```

Comment:
I would make this the same as the Tab label, like is done for the actual categories
```suggestion
          renderHeaderTitle={this.catalogEntityStore.activeCategory?.metadata.name ?? "All Categories"}
```